cherry_picker/cherry_picker.py

```diff
@@ -293,14 +293,48 @@ def get_exit_message(self, branch):
 """
 
     def get_updated_commit_message(self, cherry_pick_branch):
+        """
+        Get updated commit message for the cherry-picked commit.
+        """
+        # Get the original commit message and prefix it with the branch name
+        # if that's enabled.
         commit_prefix = ""
         if self.prefix_commit:
             commit_prefix = f"[{get_base_branch(cherry_pick_branch)}] "
-        return f"""{commit_prefix}{self.get_commit_message(self.commit_sha1)}
-(cherry picked from commit {self.commit_sha1})
+        updated_commit_message = f"{commit_prefix}{self.get_commit_message(self.commit_sha1)}"
+
+        # Add '(cherry picked from commit ...)' to the message
+        # and add new Co-authored-by trailer if necessary.
+        cherry_pick_information = f"(cherry picked from commit {self.commit_sha1})\n:"
+        # Here, we're inserting new Co-authored-by trailer and we *somewhat*
+        # abuse interpret-trailers by also adding cherry_pick_information which
+        # is not an actual trailer.
```

Comment:
Kind of a tangent, but maybe we should make it a trailer?  `Cherry-Picked-From: {self.commit_sha1}`?

Reply:
The message `cherry picked from commit ...` is the default message generated by `git` when we use `git cherry-pick -X`. So I think it should be kept that way.

Reply:
Or this can perhaps be addressed in a separate PR :)